lib/event_store_client/adapters/grpc/commands/streams/delete.rb

```diff
@@ -14,19 +11,26 @@ class Delete < Command
           use_request EventStore::Client::Streams::DeleteReq
           use_service EventStore::Client::Streams::Streams::Stub
 
-          def call(name, options: {}) # rubocop:disable Lint/UnusedMethodArgument
-            opts =
-              {
-                stream_identifier: {
-                  streamName: name
-                },
-                any: {}
-              }
-
-            service.delete(request.new(options: opts), metadata: metadata)
-            Success()
+          # @api private
+          # @see {EventStoreClient::GRPC::Client#delete_stream}
+          def call(stream_name, options:, &blk)
+            options = normalize_options(stream_name, options)
+            yield options if block_given?
```

Comment:
Why no move this 
```
 options = normalize_options(stream_name, options)
 yield options if block_given?
```

to the base class and call `super`? You are repeating it almost in every command.